test/core/python/IronPythonInfo_TestGraph.dyn

```diff
@@ -293,7 +293,7 @@
     {
       "ConcreteType": "Dynamo.Graph.Nodes.CodeBlockNodeModel, DynamoCore",
       "NodeType": "CodeBlockNode",
-      "Code": "0;\n5;\n\"2.7.8\";",
+      "Code": "0;\n5;\n\"2.7.9\";",
```

Comment:
hmm, does the test rely on this string in any way?

Reply:
Yes